### PR TITLE
Set NuGet in the `.csproj` file

### PR DESCRIPTION
**Pull request by RoyKodman**

### CODE REVIEW

File: Net.Utils.TaskManager/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Methods
+
+The `TaskManager` class provides the following methods:
+
+- `AddTask(Task task)` : Adds a single task to the manager.
+- `AddRange(IEnumerable<Task> tasks)` : Adds a collection of tasks to the manager.
+- `StartAsync()` : Starts executing the tasks asynchronously.
+- `AwaitFinish()` : Awaits the completion of all tasks.
+
+#### AddTask
```
Comment (Lomet) [inline Net.Utils.TaskManager/README.md:40]: Lets add here the use of https://github.com/The-Poolz/Net.Utils.TaskManager/blob/d6e0ec36f0c1c592338400e101ce725e642b35ee/Net.Utils.TaskManager/IAddTask.cs#L3

File: Net.Utils.TaskManager/Net.Utils.TaskManager.csproj [hunk 2]
```diff
@@ -1,9 +1,30 @@
...
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
+    
+    <PackageId>Net.Utils.TaskManager</PackageId>
+    <Title>.NET Task Manager</Title>
+    <Version>1.0.0</Version>
+    <Authors>RoyKodman</Authors>
+    <Company>ThePoolz</Company>
```
Comment (ArdenHide) [inline Net.Utils.TaskManager/Net.Utils.TaskManager.csproj:12]: Please update the company name (to 'The-Poolz') to match the organization name in NuGet

https://www.nuget.org/profiles/The-Poolz

**Review by ArdenHide — APPROVED** [review]

**Review by ArdenHide — CHANGES_REQUESTED** [review]
Also, please use exist README and LICENSE from root folder

**Review by ArdenHide — APPROVED** [review]
Everything is fine. But I still don’t see a single situation where we need the `IAddTask` interface

**Pull request merged**